sdk/java-sdk-protobuf/src/main/java/kalix/javasdk/action/Action.java

```diff
@@ -30,6 +30,34 @@
 import java.util.Optional;
 import java.util.concurrent.CompletionStage;
 
+/**
```

Comment:
I really like this comment making explicit the intent of the class. I think we could have many more and not only in the user facing APIs but the internal classes. To understand an implementation, specially if the class is complex, is necessary to understand its intention. Otherwise deviation of such intention is most likely not to be noticed.